### PR TITLE
provide shell scripts instead of travis commands

### PR DESCRIPTION
**Pull request by kinggoesgaming**
hopefully this works.

also now `apiver.txt` and `sapiver.txt` are not generated on our side unless `gradle travisFiles` is run.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ run/
 # MinigameCore
 apiver.txt
 sapiver.txt
+
```
Comment (Flibio) [inline .gitignore]: Why an extra line?

File: gradle/travis/files.gradle [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    dependsOn shadowJar
+
+    logger.lifecycle('MinigameCoreAPI: ' + project(':MinigameCoreAPI').version)
+    def apifile = new File('.travis/apiver.txt')
+    apifile.text = project(':MinigameCoreAPI').version.minus("-SNAPSHOT")
+
+    logger.lifecycle('SpongeAPI: ' + project(':MinigameCoreAPI').spongeapi)
+    def sapifile = new File('.travis/sapiver.txt')
+    sapifile.text = project(':MinigameCoreAPI').spongeapi.minus("-SNAPSHOT")
+}
```
Comment (Flibio) [inline gradle/travis/files.gradle]: Newline.

File: .travis.yml [hunk 3]
```diff
@@ -10,23 +10,19 @@ notifications:
...
 install:
-- 'wget http://stedolan.github.io/jq/download/linux64/jq'
-- 'chmod +x ./jq'
-- 'sudo cp jq /usr/bin'
+- 'chmod +x ./.travis/install.sh'
+- './.travis/install.sh'
 script:
-- ./gradlew build shadowJar
+- 'chmod +x ./gradlew'
+- './gradlew '
```
Comment (Flibio) [inline .travis.yml]: Shouldn't this call the task `travisFiles`.

File: build.gradle [hunk 4]
```diff
@@ -10,16 +10,9 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.3'
 }
 
-defaultTasks 'clean', 'build', 'shadowJar'
+defaultTasks shadowJar
```
Comment (Flibio) [inline build.gradle]: Need quotes around `shadowJar`.

File: .gitignore (hunk 1 above)
Comment (kinggoesgaming) [inline .gitignore]: must have happened while rebasing :P

File: .travis.yml (hunk 3 above)
Comment (kinggoesgaming) [inline .travis.yml]: oh no... i was doing it... then decided to finish something and yea...

**Comment by kinggoesgaming** [comment]
imma make one more change

**Pull request merged**